charts/rasa-x/templates/event-service-deployment.yaml

```diff
@@ -33,6 +33,11 @@ spec:
       # The event service uses a database, to make sure that the database is ready
       # to use we use the database migration service to check migration status.
       {{- include "initContainer.dbMigration" . |  nindent 6 }}
+      {{- if .Values.eventService.initContainers }}
+      initContainers:
```

Comment:
The problem here is that we already use a `initContainers` key in this place https://github.com/RasaHQ/rasa-x-helm/blob/master/charts/rasa-x/templates/_helpers.tpl#L267

It means if we use the db-migration service and init containers for the event service we'll have duplicate keys 🤔

Reply:
Oh dang did I add this? I didn't intend to

Reply:
Ah, I did but it was a mistake after local experimentation, removing it